painttiming.bs

```diff
@@ -62,9 +86,24 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn export>Contentful</dfn>: An element counts as contentful when one or more of the following apply:
```

Comment:
```suggestion
An element is <dfn export>contentful</dfn> when one or more of the following apply:
```
Also, where is this going to be used outside of this spec (i.e. why does it need to be export)?

Reply:
I guess it doesn't for now! I think it would be useful to have a "contentful" definition that's reused but we an do that when we get there.

Reply:
Ok! And WDYT about the suggested wording?

Reply:
Oops, just saw that this is a suggestion. Will bake it in to the PR.